README.md

```diff
@@ -3,30 +3,37 @@
 
 Contentstack is a headless CMS with an API-first approach that puts content at the centre. It is designed to simplify the process of publication by separating code from content.
 
-Currently, the CLI is in Beta and supports content management scripts through which you can perform the following tasks: 
+CLI supports content management scripts through which you can perform the following tasks:
 
 - Bulk publish content
-- Export content 
+- Export content
 - Import content
 - Clone Stack
```

Comment:
As per the doc `Compare and Merge branches` option is missing from the list

Reply:
Hey @antonyagustine,
This in intentionally not added till the branches c&m not released from the product end.
The release is planned in next week, once done we will add that.